src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -365,11 +365,14 @@ class AppStoreConnectCredentialsValidateEndpoint(ProjectEndpoint):  # type: igno
     {
         "appstoreCredentialsValid": true,
         "itunesSessionValid": true,
+        "hasDownloadsPending": false,
```

Comment:
Possibly better the number of pending downloads?  needs feedback from the UI, i.e. @priscilawebdev